### PR TITLE
[CON-208] feat: Added the provider guides for the kit connector

### PR DESCRIPTION
**Pull request by sanjay-a01**

### CODE REVIEW

File: src/images/provider-guides/kit_configure.gif
Comment (RajatPawar) [inline src/images/provider-guides/kit_configure.gif:1]: This GIF has the client secret visible in plaintext, which is not something we want to endorse. Can you please record this again without the secret showing? Thanks! I'd not show the secret in any gifs

File: src/provider-guides/kit.mdx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+Once your account is created, you'll need to create an app in Kit and obtain the following credentials from your app:
+
+- Client ID
+- Client Secret
+
+You will then use these credentials to connect your application to Ampersand.
+
+### Create an Kit Account
+
+Here's how you can sign up for an Kit account:
```
Comment (RajatPawar) [inline src/provider-guides/kit.mdx:24]: Same as the other PR, let's skip this section

File: src/provider-guides/kit.mdx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+- Go to the [Kit Sign Up page](https://app.kit.com/users/signup?plan=1k).
+- Sign up using your preferred method.
+
+### Creating an Kit App
```
Comment (RajatPawar) [inline src/provider-guides/kit.mdx:29]: Can you fix all occurrences of `an Kit` to `a Kit`?

File: src/provider-guides/kit.mdx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+Follow the steps below to create an Kit app and add the Ampersand redirect URL.
+
+1. Log in to your [Kit](https://app.kit.com/users/login) account.
+
+2. Click **Profile** icon, then click on **Settings** and go to the **Developer** in settings.
```
Comment (RajatPawar) [inline src/provider-guides/kit.mdx:35]:
```suggestion
2. Click **Profile** icon, then click on **Settings** and go to **Developer**
```

File: src/provider-guides/kit.mdx [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+3. Click **Create a new app**.
```
Comment (RajatPawar) [inline src/provider-guides/kit.mdx:37]:
```suggestion
3. Click on **Create a new app**.
```

File: src/provider-guides/kit.mdx [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+4. In the **Create a new app** pop-up window, enter the **App name**. 
+
+5. You can also add the below section
```
Comment (RajatPawar) [inline src/provider-guides/kit.mdx:41]:
```suggestion
5. You can also add -
```

File: src/provider-guides/kit.mdx [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+- **Logo**
+- **Website URL**
+- **Support URL**
+- **Knowledge base URL**
+- **Privacy policy URL**
+- **Images**
+- **Access required**
+- **Categories**
+
+5. Click **Save**. Then Click **Configure**.
```
Comment (RajatPawar) [inline src/provider-guides/kit.mdx:53]:
```suggestion
5. Click **Save**. Next, you will need to **Configure** the app.
```

File: src/provider-guides/kit.mdx [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+    ![Kit App Creation](/images/provider-guides/kit_OAuth.gif)
+
+6. Enter the Authorization URL: `https://app.kit.com/oauth/authorize` in the **Authorization URL** section.
```
Comment (RajatPawar) [inline src/provider-guides/kit.mdx:57]: Does kit need us to give the OAuth URL?

File: src/provider-guides/kit.mdx [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+3. Select **Provider Apps**.
+
+4. Select _Kit_ from the **Provider** list.
+
+5. Enter the previously obtained _Client ID_ in the **Client ID** field and the _Client Secret_ in the **Client Secret** field.
+
+    ![Ampersand Integration](/images/provider-guides/kit.gif)
+
+6. Click **Save Changes**.
```
Comment (RajatPawar) [inline src/provider-guides/kit.mdx:81]:
```suggestion
```

**Review by RajatPawar — CHANGES_REQUESTED** [review]
Minor changes, great work on the GIFs! Can you also run this through ChatGPT to make sure any grammatical errors are fixed, and to rephrase all sentences in a manner appropriate for a technical document?

File: src/provider-guides/kit.mdx (hunk 7 above)
Comment (sanjay-a01) [reply src/provider-guides/kit.mdx:57]: Yeah! we need to give OAuth URL.

File: src/provider-guides/kit.mdx [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+- **Access required**
+- **Categories**
+
+5. Click **Save**. Next, you will need to **Configure** the app.
+
+    ![Kit App Creation](/images/provider-guides/kit_OAuth.gif)
+
+6. Enter the Authorization URL: `https://app.kit.com/oauth/authorize` in the **Authorization URL** section.
+
+7. Enter the Redirect URIs: `https://api.withampersand.com/callbacks/v1/oauth` in the **Redirect URIs** section.
```
Comment (RajatPawar) [inline src/provider-guides/kit.mdx:59]:
```suggestion
7. Enter `https://api.withampersand.com/callbacks/v1/oauth` in the **Redirect URIs** section.
```

**Review by RajatPawar — APPROVED** [review]

**Pull request merged**